src/test/java/com/crowdin/cli/commands/actions/DownloadActionTest.java

```diff
@@ -198,25 +200,12 @@ public void testProjectOneFittingFile_WithExportApprovedOnly_WithSkipUntranslate
                 }));
 
         NewAction<PropertiesWithFiles, ProjectClient> action =
-            new DownloadAction(files, false, null, false, null, false, false, false, false);
+            new DownloadAction(files, false, null, false, null, false, false, false, false, true);
         action.act(Outputter.getDefault(), pb, client);
 
         verify(client).downloadFullProject();
-        verify(client).startBuildingTranslation(eq(buildProjectTranslationRequest));
```

Comment:
Here is also something is wrong. We can't delete such a part of the test. It is checking an important part of code.

This means that something is broken and there shouldn't be any runtime exceptions.